app/internal/page/data.go

```diff
@@ -141,7 +139,13 @@ func DecodeAddress(s string) *place.Address {
 }
 
 func (l *Lpa) IdentityConfirmed() bool {
```

Comment:
Seeing as we've now switched from `Person` to `Donor` is it worth scoping this func name to `DonorIdentityConfirmed` for clarity?